service/controller/app/v1/key/key.go

```diff
@@ -35,6 +35,10 @@ func SecretNamespace(customResource v1alpha1.App) string {
 	return customResource.Spec.Config.Secret.Namespace
 }
 
+func AppStatus(customResource v1alpha1.App) string {
+	return customResource.Status.Status
```

Comment:
`Status.Status` is not cool. Can we improve that naming? What kind of status is this about? What are the possible values?

Reply:
Can we then have `Release.Status`? I think the structure makes total sense. Like it a lot in the VOO specs.

Reply:
Yes I think that works.

Reply:
Should I wait until it merges into apiextention? or we can have it later?

Reply:
@tomahawk28 We can deal with that separately IMO. I'll take it as I want to keep the chart and app CR statuses aligned. I've just been busy with other things.

But I don't see any changes related to storing the g8s client in the context. Have you looked into that?

Reply:
Spec PR is up that includes the status changes. Please discuss there and I'll take care once we're happy.

https://github.com/giantswarm/giantswarm/pull/5240

Reply:
The CRD changes are separate but AppStatus should be ordered in the file.